features/background-size.yml.dist

```diff
@@ -3,14 +3,14 @@
 
 status:
   baseline: high
-  baseline_low_date: 2020-07-28
-  baseline_high_date: 2023-01-28
+  baseline_low_date: ≤2020-07-28
+  baseline_high_date: ≤2023-01-28
   support:
     chrome: "21"
     chrome_android: "25"
     edge: "12"
     firefox: ≤72
-    firefox_android: "79"
+    firefox_android: ≤79
```

Comment:
Newly ranged. Not a regression per se, but uncertainty has been added.

Reply:
I just opened https://github.com/mdn/browser-compat-data/pull/25356 to replace this range with a specific version.